src/worker/exporters/node_exporter.py

```diff
@@ -254,7 +264,9 @@ def init_config(job_id, port=None):
         'job_id': job_id,
         'fieldFiles': {},
         'counter': {},
+        'event_timestamp': datetime.datetime.now().strftime("%b %d %H:%M:%S")
```

Comment:
No year?

Reply:
correct it follows the syslog format of: May 21 00:00:06